app.js

```diff
@@ -17,7 +17,7 @@ function handleGuess(guess) {
     score(guess);
```

Comment:
This function call doesn't need to be here. You are already calling the function (almost) properly down on line 20, so having this function up here as well is causing buggy scoring functionality.

Reply:
Hey James,
Thank you for your feedback! I fixed the function and removed the extra line of code that was causing the bug.